src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -58,19 +58,19 @@ public void testDot()
         checkConfig.addAttribute("option", "NL");
         checkConfig.addAttribute("tokens", "DOT");
         final String[] expected = {
-            "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
+            "29:10: " + getCheckMessage(MSG_LINE_NEW, "."),
```

Comment:
@rnveach 
Input was not copied. It was changed in order to become more specific to test content. For example, if we test NL option the input file should contains positive and negative test cases with comments for new line. This will make the input file shorter and it content will become more readable. We already have input files which are hard to read because they contain to many unrelated code examples. I think that each input file should be as short as possible, each UT too.